src/main/scala/is/hail/stats/BaldingNicholsModel.scala

```diff
@@ -6,15 +6,15 @@ import breeze.linalg.{DenseVector, sum}
 import org.apache.spark.SparkContext
 import org.apache.commons.math3.random.JDKRandomGenerator
 import is.hail.annotations.Annotation
-import is.hail.expr.{TArray, TDouble, TInt, TStruct}
+import is.hail.expr.{TArray, TDouble, TInt, TString, TStruct}
 import is.hail.utils._
 import is.hail.variant.{Genotype, Variant, VariantDataset, VariantMetadata}
 
 object BaldingNicholsModel {
 
   def apply(sc: SparkContext, nPops: Int, nSamples: Int, nVariants: Int,
     popDistArrayOpt: Option[Array[Double]], FstOfPopArrayOpt: Option[Array[Double]],
-    seed: Int, nPartitionsOpt: Option[Int], root: String): VariantDataset = {
+    seed: Int, nPartitionsOpt: Option[Int], af_dist: Distribution, root: String): VariantDataset = {
```

Comment:
Did we decide against changing the type of `af_dist` to `RandBasis => Rand[Double]`?